devfile.yaml

```
@@ -13,13 +13,23 @@ components:
   - name: m2
     volume: {}
 commands:
-  - id: build
+  - id: buildsources
     exec:
-      label: "1. Build"
+      label: "1. Build sources"
       component: tools
       workingDir: ${PROJECT_SOURCE}
       commandLine: |
         mvn clean install -DskipTests
```

```suggestion
        mvn clean install -V -e -Pfast -DskipTests -Dskip-validate-sources -Denforcer.skip=true
```

Because the enforcer/validater tends to fail because of not updated license headers, this is a safer solution, and how we buildit downstream.